contrib/aws/aws-sdk-go/aws/aws.go

```diff
@@ -20,11 +20,13 @@ import (
 )
 
 const (
-	tagAWSAgent      = "aws.agent"
-	tagAWSOperation  = "aws.operation"
-	tagAWSRegion     = "aws.region"
-	tagAWSRetryCount = "aws.retry_count"
-	tagAWSRequestID  = "aws.request_id"
+	tagAWSAgent         = "aws.agent"
+	tagAWSOperation     = "aws.operation"
+	tagAWSRegion        = "aws.region"
+	tagAWSRetryCount    = "aws.retry_count"
+	tagAWSRequestID     = "aws.request_id"
+	SendHandlerName     = "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws/handlers.Send"
+	CompleteHandlerName = "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws/handlers.Complete"
```

Comment:
These need comments documenting what they are.

Reply:
Sure, thanks. Done, happy to revise the language if desired.

Reply:
Looks good, thanks.